src/commands/deployWorkspaceProject/deployWorkspaceProjectSettings.ts

```diff
@@ -39,3 +42,25 @@ export async function setDeployWorkspaceProjectSettings(rootFolder: WorkspaceFol
         await settingUtils.updateWorkspaceSetting(`${deployWorkspaceProjectPrefix}.${key}`, settings[key], settingsPath, ConfigurationTarget.WorkspaceFolder);
     }
 }
+
+export function displayDeployWorkspaceProjectSettingsOutput(settings: DeployWorkspaceProjectSettings): void {
+    if (hasAllDeployWorkspaceProjectSettings(settings)) {
+        // Skip, more detailed logs will come when we confirm whether or not the resources were found
+    } else if (hasAtLeastOneDeployWorkspaceProjectSetting(settings)) {
+        ext.outputChannel.appendLog(localize('resourceSettingsIncomplete', 'Scanned and found incomplete containe app resource settings at "{0}".', relativeSettingsFilePath));
+    } else {
+        ext.outputChannel.appendLog(localize('noWorkspaceSettings', 'Scanned and found no matching resource settings at "{0}".', relativeSettingsFilePath));
```

Comment:
```suggestion
        ext.outputChannel.appendLog(localize('noWorkspaceSettings', 'Unable to find container app workspace settings at "{0}".', relativeSettingsFilePath));
```

Reply:
Kind of a minor nit, but using the word `unable` gives me the feeling as a user that something isn't working properly.  But I think it's fine to not have any workspace settings yet, so I wonder if there's a more neutral way to say that.  Maybe just "Found no container app deploy workspace settings at...`?

Reply:
Yeah, that's fine with me.